app/_hub/seifchen/kong-path-whitelist/index.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Kong Path White List
+publisher: Seifchen
+
+categories:
+  - security
+
+type: plugin
+
+desc: Determine if the path is in the path whitelist, if not, return 403
+
+description: |
+  Add Kong Path White List to a Service or a Route or consumer with paths. The plugin will check the path is in the path whitelist, if not return 403.
+
+support_url: https://github.com/seifchen/kong-path-whitelist/issues
+
+source_url: https://github.com/seifchen/kong-path-whitelist
+
+license_type: Apache-2.0
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+      - 1.3.x
+      - 1.2.x
+    incompatible:
```

Comment:
Can you remove the incompatible section? We're going to stop including them going forward.

Reply:
done